ec2/snapshots.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 func (e *Ec2) ListSnapshots(ctx context.Context, org string, per int64, next *string) ([]map[string]*string, *string, error) {
+	//TODO: need to replace this function with DescribeSnapshots
```

Comment:
Maybe we can create a separate task to replace this, we'll need to add an orchestrator function